src/actions/sidebar.action.js

```diff
@@ -39,3 +56,25 @@ export function setEditingStoryId(storyId) {
 export function updateStoryNameInput(storyName) {
   return dispatch => dispatch(storyNameInputUpdated(storyName))
 }
+
+export function updatePoisInput(pois) {
```

Comment:
this name is a little confusing since "input" usually refers to a form input. i believe this is actually referring to the list of pois that are selected? if so, this function should be renamed to reflect that so that someone else reading the code elsewhere doesn't assume this updates a form input

Reply:
also change the corresponding action too